assets/sass/components/global/_googlesitekit-header.scss

```diff
@@ -90,8 +90,8 @@
 			.mdc-button__icon,
 			.mdc-button__icon--image {
 				// increase default button image size
-				height: 28px;
-				width: 28px;
+				height: 24px;
```

Comment:
@hussain-t As per the AC, the width and height of the image should be `28px`. You can set the padding to the parent button (`.googlesitekit-user-selector button`) to `4px` so that the total width is `36px` (28 + 4 + 4).